scroll-kit.coffee

```diff
@@ -330,6 +339,9 @@ initialize_sticky = (node) ->
   node.isFixed = data.fixed or (el.css('position') is 'fixed')
   node.placeholder = placeholder(node) if update_sticky(node)
 
+  if node.data.fit
+    node.el.on('DOMMouseScroll mousewheel', preventScroll)
```

Comment:
Maybe we should return to this.
Current implementation only works for element first parent-children relationship _(`el.parentElement` once; think in an `e.target` that's not first children like an anchor inside a span, inside a div, inside the sticky element)_, one solution could be search for sticky parent recursively but I'm not sure if that's a better approach than this.